server/src/com/cloud/network/NetworkServiceImpl.java

```diff
@@ -3085,9 +3085,10 @@ public GuestVlan dedicateGuestVlanRange(DedicateGuestVlanRangeCmd cmd) {
         // Verify account is valid
         Account vlanOwner = null;
         if (projectId != null) {
-            if (accountName != null) {
-                throw new InvalidParameterValueException("accountName and projectId are mutually exclusive");
-            }
+            //accountName and projectId are mutually exclusive
```

Comment:
@nitin-maharana So as I understand in order to avoid making a breaking API change you are simply ignoring account and giving priority to project if both are specified.